<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialogUI</class>
 <widget class="QDialog" name="UserDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Info</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="mEmployeeGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>448</width>
       <height>155</height>
      </size>
     </property>
     <property name="title">
      <string>Employee Information</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="mPixmapLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>125</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>125</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>125</width>
           <height>125</height>
          </size>
         </property>
         <property name="toolTip">
          <string comment="Employee picture"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout">
          <property name="margin">
           <number>1</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="4" column="0">
           <widget class="QLabel" name="textLabel4">
            <property name="text">
             <string>Jabber Id:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="mJid"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="mFirst"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1">
            <property name="text">
             <string>First Name : </string>
            </property>
            <property name="buddy">
             <cstring></cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="textLabel3">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="textLabel7">
            <property name="text">
             <string>Middle Name : </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="textLabel2">
            <property name="text">
             <string>Last Name :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="mMiddle"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="mEmail"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="mLast"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mAdminGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Admin Information</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel5">
          <property name="text">
           <string>User Name : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mUserName"/>
        </item>
        <item>
         <widget class="QLabel" name="textLabel6">
          <property name="text">
           <string>Password : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel8">
          <property name="text">
           <string>Assigned Hosts:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mAssignedHostsEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mAssignedHostsButton">
          <property name="text">
           <string>Change Hosts...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="mShellCheck">
          <property name="text">
           <string>Allow Shell Access</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="mNotifyJabberCheck">
          <property name="text">
           <string>Jabber  Message Forwarding</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="mEditGroups">
          <property name="minimumSize">
           <size>
            <width>212</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Edit Groups</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="mNotifyEmailCheck">
          <property name="text">
           <string>Email Message Forwarding</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>155</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOKButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mFirst</tabstop>
  <tabstop>mMiddle</tabstop>
  <tabstop>mLast</tabstop>
  <tabstop>mJid</tabstop>
  <tabstop>mEmail</tabstop>
  <tabstop>mUserName</tabstop>
  <tabstop>mPassword</tabstop>
  <tabstop>mShellCheck</tabstop>
  <tabstop>mEditGroups</tabstop>
  <tabstop>mNotifyEmailCheck</tabstop>
  <tabstop>mNotifyJabberCheck</tabstop>
  <tabstop>mOKButton</tabstop>
  <tabstop>mCancelButton</tabstop>
  <tabstop>mNotifyEmailCheck</tabstop>
  <tabstop>mNotifyJabberCheck</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>UserDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOKButton</sender>
   <signal>clicked()</signal>
   <receiver>UserDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
